### PR TITLE
Add Haskell support

### PR DESCRIPTION
**Pull request by evanrelf**
## Description

Added a Haskell section to show the version of GHC being used by Stack. Imitates [spaceship-prompt's implementation](https://github.com/denysdovhan/spaceship-prompt/blob/master/sections/haskell.zsh).

Also fixed a typo in `docs/Options.md` (`s/SHIP/FISH`).

## Motivation and Context

Brings parity between spaceship-prompt and spacefish regarding Haskell support.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Screenshots (if appropriate):
screen shot 2018-08-16 at 11 24 42 pm

## How Has This Been Tested?

- [x] I have tested using **MacOS**
- [ ] I have tested using **Linux**

## Checklist:

- [x] I have checked that no other PR duplicates mine
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have updated the tests accordingly.

### CODE REVIEW

File: docs/Options.md [hunk 1]
```diff
@@ -85,7 +85,7 @@ Hostname should only be shown while you are connected to another machine using S
 | `SPACEFISH_HOST_SHOW` | `true` | Show host section (true/false) |
 | `SPACEFISH_HOST_SHOW_FULL` | `false` | Show full hostname section (true/false) |
 | `SPACEFISH_HOST_PREFIX` | `at·` | Prefix before the hostname |
-| `SPACEFISH_HOST_SUFFIX` | `$SPACESHIP_PROMPT_DEFAULT_SUFFIX` | Suffix after the hostname |
+| `SPACEFISH_HOST_SUFFIX` | `$SPACEFISH_PROMPT_DEFAULT_SUFFIX` | Suffix after the hostname |
```
Comment (matchai) [inline docs/Options.md:88]: Good catch! 👍

File: docs/Options.md [hunk 2]
```diff
@@ -195,3 +195,16 @@ Ruby section is shown only in directories that contain `Gemfile`, or `Rakefile`,
...
+
+Ruby section is shown only in directories that contain `stack.yaml`.
+
+| Variable | Default | Meaning |
+| :------- | :-----: | ------- |
+| `SPACEFISH_HASKELL_SHOW` | `true` | Show Haskell section |
+| `SPACEFISH_HASKELL_PREFIX` | `$SPACEFISH_PROMPT_DEFAULT_PREFIX` | Prefix before Haskell section |
+| `SPACEFISH_HASKELL_SUFFIX` | `$SPACEFISH_PROMPT_DEFAULT_SUFFIX` | Suffix after Haskell section |
+| `SPACEFISH_HASKELL_SYMBOL` | `"λ "` | Character to be shown before Haskell version |
+| `SPACEFISH_HASKELL_COLOR` | `"red"` | Color of Haskell section |
```
Comment (matchai) [inline docs/Options.md]: Just a quick note about the default values here.
I would remove the quotes within the backticks to keep them consistent with the rest of the docs.
Also, please replace the space after the lambda with a mid-dot: `·`

**Review by matchai — APPROVED** [review]
Thank you so much for your contribution @evanrelf! 😄
Just a small nitpick about documentation, but otherwise your PR looks great!

**Comment by evanrelf** [comment]
Happy to help! 😄

**Pull request merged**